docs/package.json

```diff
@@ -11,72 +10,90 @@
+		"lint": "concurrently npm:eslint npm:prettier",
+		"lint:fix": "npm run eslint:fix && npm run prettier:fix",
 		"prettier": "prettier --check . --cache --ignore-path ../.prettierignore",
 		"prettier:fix": "prettier --write . --cache --ignore-path ../.prettierignore",
-		"start": "hugo server",
-		"start-with-azure-emulation": "swa start public --api-location api"
+		"rebuild": "npm run clean && npm run build",
+		"serve": "docusaurus serve",
+		"serve-with-azure-emulation": "swa start build --api-location api",
+		"prestart": "npm run generate-versions",
+		"start": "docusaurus start",
+		"pretest": "playwright install --with-deps",
+		"test": "playwright test"
+	},
+	"browserslist": {
+		"production": [
+			">0.5%",
+			"not dead",
+			"not op_mini all"
+		],
+		"development": [
+			"last 3 chrome version",
+			"last 3 firefox version",
+			"last 5 safari version"
+		]
 	},
 	"dependencies": {
+		"@docusaurus/core": "^3.5.2",
+		"@docusaurus/module-type-aliases": "^3.5.2",
+		"@docusaurus/plugin-content-docs": "^3.5.2",
+		"@docusaurus/preset-classic": "^3.5.2",
+		"@docusaurus/theme-common": "^3.5.2",
+		"@docusaurus/theme-mermaid": "^3.5.2",
+		"@docusaurus/tsconfig": "^3.5.2",
+		"@docusaurus/types": "^3.5.2",
+		"@rushstack/node-core-library": "^5.9.0",
+		"clsx": "^2.0.0",
+		"docusaurus-lunr-search": "^3.5.0",
+		"docusaurus-plugin-sass": "^0.2.5",
+		"lunr": "^2.3.9",
+		"prism-react-renderer": "^2.3.0",
+		"react": "^18.0.0",
+		"react-dom": "^18.0.0"
+	},
+	"devDependencies": {
 		"@azure/static-web-apps-cli": "^2.0.1",
 		"@fluid-tools/api-markdown-documenter": "^0.17.1",
 		"@fluid-tools/markdown-magic": "file:../tools/markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
-		"@rushstack/node-core-library": "^4.0.2",
-		"@vscode/codicons": "0.0.35",
+		"@fluidframework/eslint-config-fluid": "^5.4.0",
+		"@playwright/test": "^1.48.0",
+		"@types/node": "^22.8.6",
 		"chalk": "^5.3.0",
-		"concurrently": "^8.2.2",
-		"deepdash": "^5.3.9",
+		"concurrently": "^9.0.1",
+		"cross-env": "^7.0.3",
 		"dill-cli": "^0.1.3",
+		"dotenv-cli": "^7.4.2",
+		"eslint": "~8.55.0",
+		"eslint-config-prettier": "~9.1.0",
 		"fs-extra": "^11.2.0",
-		"glob": "^10.3.12",
-		"hast-util-format": "^1.1.0",
-		"hast-util-to-html": "^9.0.3",
-		"http-server": "^14.1.1",
-		"hugo-extended": "^0.113.0",
-		"js-yaml": "^4.1.0",
-		"linkcheck-bin": "3.0.0-1",
-		"markdownlint-cli2": "^0.12.1",
-		"markdownlint-rule-emphasis-style": "^1.0.1",
-		"markdownlint-rule-github-internal-links": "^0.1.0",
-		"markdownlint-rule-helpers": "^0.25.0",
-		"prettier": "~3.2.5",
-		"prettier-plugin-go-template": "^0.0.15",
-		"rimraf": "^5.0.5",
-		"start-server-and-test": "^2.0.3"
+		"playwright": "^1.48.0",
+		"prettier": "^3.2.5",
+		"rimraf": "^6.0.1",
+		"typescript": "~5.5.2"
 	},
-	"packageManager": "pnpm@8.15.8+sha512.d1a029e1a447ad90bc96cd58b0fad486d2993d531856396f7babf2d83eb1823bb83c5a3d0fc18f675b2d10321d49eb161fece36fe8134aa5823ecd215feed392",
```

Comment:
I think we want to keep this? Partly for consistency with the rest of the repo and to have a defined version. Partly because I think the lockfile generated by pnpm 9 is not supported by Component Governance yet.